test-e2e/core/import.test.ts

```diff
@@ -46,6 +46,6 @@ test('should load and migrate .nmrium data from version 2 to version 3', async (
   await nmrium.clickPanel('Filters');
 
   await expect(
-    nmrium.page.locator('data-test-id=filters-table >> text=Apodization'),
+    nmrium.page.locator('_react=FilterPanelInner >> text=Apodization'),

```

Comment:
Does it work with `_react=FilterPanel` ? We should avoid "inner" or "internal" things if possible.

Reply:
it works thanks